src/effects/backends/builtin/lvmixeqbase.h

```diff
@@ -22,8 +22,8 @@ class LVMixEQEffectGroupState : public EffectState {
   public:
     explicit LVMixEQEffectGroupState(const mixxx::EngineParameters& engineParameters)
             : EffectState(engineParameters),
-              m_oldLow(1.0),
-              m_oldMid(1.0),
+              m_oldLow(0.0),
+              m_oldMid(0.0),
```

Comment:
In our LV mix, the high part is carrying the dry stream. 
So this is an obvious fix to ramp from dry after initializing.